src/Plugin/Condition/DataListContains.php

```diff
@@ -35,14 +35,11 @@
  * @todo: Add info alter
  */
 class DataListContains extends RulesConditionBase {
-
+  
   /**
-   * {@inheritdoc}
+   * Evaluate whether the list has the item
    */
```

Comment:
I'm declaring $list as an array, since data type 'list' stores its data internally as an array.  Not sure in general how to translate some TypedData types into PHP types for this purpose.